src/styles/styles.js

```diff
@@ -2174,7 +2174,7 @@ const styles = {
     },
 
     iouPreviewBoxLoading: {
-        minHeight: 47,
+        minHeight: 94,
```

Comment:
@syedsaroshfarrukhdot Add a comment in the code here, explaining why it was 94, thanks

Reply:
Just added the comment. Thanks